source/administration/changelog.md

```diff
@@ -6,7 +6,10 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.25 - [ESR](https://docs.mattermost.com/administration/release-definitions.html#extended-support-release-esr)
 
-**Release day: 2020-07-16**
+- **v5.25.1, released **
+  - Fixed an issue where LDAP sync sent cache clear messages in a tight loop. [MM-27114](https://mattermost.atlassian.net/browse/MM-27114)
```

Comment:
Unsure if the description needs any more details.

Reply:
I will send an updated description once I am able to reproduce the issue more clearly.

Reply:
There are 3 fixes now:
- Smoothen the DB query load while syncing teams and channel roles by fetching data in batches.
- Fix a bug in pagination which would query more data redundantly.
- Throttle network traffic by implementing bounded concurrency.